packages/modals/src/styled/StyledClose.ts

```diff
@@ -40,6 +40,12 @@ const colorStyles = (props: ThemeProps<DefaultTheme>) => {
   `;
 };
 
+export const BASE_MULTIPLIERS = {
+  top: 2.5,
+  side: 6.5,
+  size: 10
```

Comment:
It's not clear to me how these values were derived. Could you clarify where these values come from or how they were determined?

Reply:
Is the `BASE_MULTIPLIERS` juice extraction worth the squeeze? Maybe just me, but I feel like direct `props.theme.space.base` calculations combined with meaningful comments would be easier to follow.

Reply:
@hzhu these values were extracted from what preexisted for the CSS properties (eg `top: props.theme.space.base * 2.5`) and mapped to this const variable.

@jzempel I think there are benefits to this, without much squeezing IMO. it allows us to update the value in one place if we want to amend in the future, and the imports do tether the values and can be traced back in the code, showing the relationship with the values which is easier to read. I think a part of that would be lost without it.

If y'all don't think it's worth it and prefer literal values, fine by me.

Reply:
I like having connective tissue – we should probably have more – it's just this construct feels like it takes more comment to explain than can be derived from the code. YMMV, so fine w me if you go with your gut.